components/dashboard/src/settings/SelectIDE.tsx

```diff
@@ -0,0 +1,187 @@
+/**
+ * Copyright (c) 2022 Gitpod GmbH. All rights reserved.
+ * Licensed under the GNU Affero General Public License (AGPL).
+ * See License-AGPL.txt in the project root for license information.
+ */
+
+import { IDEOption, IDEOptions } from "@gitpod/gitpod-protocol/lib/ide-protocol";
+import { useContext, useEffect, useState } from "react";
+import InfoBox from "../components/InfoBox";
+import SelectableCardSolid from "../components/SelectableCardSolid";
+import Tooltip from "../components/Tooltip";
+import { getGitpodService } from "../service/service";
+import { UserContext } from "../user-context";
+import CheckBox from "../components/CheckBox";
+
+export interface SelectIDEProps {
+    showLatest?: boolean;
+}
+
```

Comment:
`migrationIDESettings` should move to this file too.

And will do this https://github.com/gitpod-io/gitpod/pull/8783#discussion_r845141061 in the future